k8s/upgrades/0.4.0-0.5.0/replica.patch.yml

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
can we name this file as replica.patch.tpl.yml

Reply:
Done